### PR TITLE
Pin commonmark dependency to 0.5.4

### PR DESCRIPTION
**Pull request by jasongrout**
See https://github.com/rtfd/recommonmark/commit/7ca5247b342e755ce5294d43b5fef68afd90028e

This depends on the commonmark feedstock building 0.5.4: conda-forge/commonmark-feedstock#6

### CODE REVIEW

**Comment by jasongrout** [comment]
See also https://github.com/conda-forge/commonmark-feedstock/pull/5 for building commonmark 0.5.4.

**Comment by CJ-Wright** [comment]
@jasongrout do you want to be a maintainer? (also just ping me when everything is ready to go)

**Comment by jasongrout** [comment]
> @jasongrout do you want to be a maintainer? (also just ping me when everything is ready to go)

Thanks. I don't particularly want to be a maintainer, but I'm happy to contribute PRs to get my repos working :).

**Comment by CJ-Wright** [comment]
Fair enough

**Comment by jasongrout** [comment]
@CJ-Wright - I think this is ready to merge.

**Comment by jasongrout** [comment]
ping @CJ-Wright - I think this is ready to merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ about:
   home: https://github.com/rtfd/recommonmark
   license: MIT
   license_family: MIT
-#  license_file: LICENSE.md
+  # license_file: license.md # Their license.md is not distributed in their pypi package.
```
Comment (CJ-Wright) [inline recipe/meta.yaml:36]: Can you make an issue on this at the source?

**Review by CJ-Wright — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (jasongrout) [reply recipe/meta.yaml:36]: I think it is fixed in their master: https://github.com/rtfd/recommonmark/blob/master/MANIFEST.in#L1

Comment (CJ-Wright) [reply recipe/meta.yaml:36]: Brilliant!

**Pull request merged**